src/whisper_at_train/run_combined_training.sh

```
@@ -6,50 +6,98 @@
+# Validate pretrained model exists
+if [ ! -f "${pretrained_model}" ]; then
+    echo "Pretrained model not found: ${pretrained_model}"
+    exit 1
+fi
+
 # Get current timestamp in YYMMDDHHMM format
 timestamp=$(date +%y%m%d%H%M)
 
+# Create experiment directory
 exp_dir=./exp/combined-ft-${dataset}-${model}-${model_size}-${lr}-${lrscheduler_start}-${lrscheduler_decay}-ep${epoch}-bs${batch_size}-lda${lr_adapt}-ls${label_smooth}-mix${mixup}-${freqm}-${timem}-${timestamp}
 mkdir -p $exp_dir
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)